### PR TITLE
feat: [FFM-4481]: update audit log lists

### PR DESCRIPTION
**Pull request by crystal-fong**
### Summary

The following patch instructions are updated to a more user-friendly version:

`addClause` = `A clause was added`
`addToExcludeList` = `'{{target}} added to excluded Targets'`
`removeFromExcludeList` = `'{{target}} removed from excluded Targets'`
`addToIncludeList` = `'{{target}} added to included Targets'`
`removeFromIncludeList` = `'{{target}} removed from included Targets'`

image

#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11344)

**Comment by crystal-fong** [comment]
retrigger titlecheck

**Review by laura-mannus — DISMISSED** [review]
lovely

File: src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx [hunk 1]
```diff
@@ -89,6 +89,26 @@ describe('AuditLogsList', () => {
...
       switch (trail.instructionSet[0]?.Kind) {
+        case 'addClause':
+          expect(screen.queryByText('cf.auditLogs.events.addClause')).toBeInTheDocument()
+          break
+
+        case 'addToIncludeList':
+          expect(screen.queryByText('cf.auditLogs.events.addToIncludeList')).toBeInTheDocument()
+          break
+
+        case 'addToExcludeList':
```
Comment (knagurski) [inline src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:100]: Looks like this is the only clause of the new ones that will be exercised. The other 4 aren't activated by the test data

**Review by knagurski — COMMENTED** [review]
Really nice work. Just a few missing test cases

File: src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx (hunk 1 above)
Comment (crystal-fong) [reply src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:100]: how were you able to check this?

Comment (knagurski) [reply src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:100]: I didn't see any updates to the test data, so I fired up the test locally and set a breakpoint on each of the new `expect` lines and it only hit one of them. A search in the test data confirmed things.

Comment (crystal-fong) [reply src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:100]: BUSTED. 
I've updated the test data and whittled the coverage even more
image

File: src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx [hunk 2]
```diff
@@ -183,6 +208,19 @@ describe('AuditLogsList', () => {
...
+          expect(screen.queryByText('cf.auditLogs.events.setFeatureFlagStateOn')).toBeInTheDocument()
+          break
+
+        case 'addTag':
+          expect(screen.queryAllByText('cf.auditLogs.events.tagUpdated')[0]).toBeInTheDocument()
+          break
+
+        case 'updateTag':
+          expect(screen.queryAllByText('cf.auditLogs.events.tagUpdated')[1]).toBeInTheDocument()
+          break
```
Comment (knagurski) [inline src/modules/75-cf/components/AuditLogs/__tests__/AuditLogsList.test.tsx:223]: Very thorough! We don't even support tags, but if/when we do, this'll already be there.

**Review by knagurski — APPROVED** [review]
🔥

**Review by laura-mannus — APPROVED** [review]
Your PRs are getting so good, so impressed with the quality

**Pull request merged**